lib/create-cma-http-client.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * @packageDocumentation
+ * @hidden
+ */
+
+import axios from 'axios'
+// @ts-ignore
+import {createHttpClient, getUserAgentHeader} from 'contentful-sdk-core'
+import cloneDeep from 'lodash/cloneDeep'
+
+import { Agent as httpAgent } from 'http'
+import { Agent as httpsAgent } from 'https'
+import { AxiosProxyConfig } from 'axios'
+
+export type ClientParams = {
```

Comment:
there's some other client params that weren't reflected inthe jsdocs 😅  (but they're in the readme! hah)

* requestLogger
* responseLogger
* basePath

https://github.com/contentful/contentful-sdk-core/blob/master/lib/create-http-client.js

Reply:
@phoebeschmidt I added requestLogger and reponseLogger. There no JSDoc in `contentful-sdk-core` about basePath, so I'm not sure how to describe the purpose. Are we sure that we want to expose it here?

Reply:
you're right maybe let's not expose it